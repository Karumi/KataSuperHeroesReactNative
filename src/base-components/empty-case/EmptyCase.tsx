import * as React from "react";
import { StyleSheet, Text, View } from "react-native";

class EmptyCase extends React.Component {
    public render() {
        return (
            <View style={style.container}>
                <Text style={style.text} >¯\_(ツ)_/¯</Text>;
            </View>
        );
    }
}

const style = StyleSheet.create({
    container: {
        flex: 1,
        justifyContent: "center",
        alignItems: "center",
    },
    text: {
        color: "white",
    },
});

export default EmptyCase;
