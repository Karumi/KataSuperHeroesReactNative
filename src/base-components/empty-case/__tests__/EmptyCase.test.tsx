import * as React from "react";
import * as renderer from "react-test-renderer";

import EmptyCase from "../EmptyCase";

it("renders with the default configuration", () => {
    const component = renderer
        .create(<EmptyCase />)
        .toJSON();
    expect(component).toMatchSnapshot();
});
