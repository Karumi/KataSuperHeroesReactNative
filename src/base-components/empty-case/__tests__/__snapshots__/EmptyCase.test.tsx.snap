// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with the default configuration 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
      }
    }
  >
    ¯\\_(ツ)_/¯
  </Text>
  ;
</View>
`;
